lib/reporters/html-cov.js

```diff
@@ -17,25 +18,31 @@ exports = module.exports = HTMLCov;
  *
  * @api public
  * @param {Runner} runner
+ * @param {Object} options Runner optional options
  */
-function HTMLCov(runner) {
+function HTMLCov(runner, options) {
+  JSONCov.call(this, runner, options, false);
+
   var jade = require('jade');
   var file = join(__dirname, '/templates/coverage.jade');
   var str = readFileSync(file, 'utf8');
   var fn = jade.compile(str, { filename: file });
   var self = this;
 
-  JSONCov.call(this, runner, false);
-
   runner.on('end', function() {
-    process.stdout.write(fn({
+    self.write(fn({
       cov: self.cov,
       coverageClass: coverageClass
     }));
   });
 }
 
 /**
+ * Inherit from `JSONCov.prototype`.
+ */
+inherits(HTMLCov, JSONCov);
```

Comment:
I can see in the CHANGELOG that they were removed in v3. Issue about that: #2356.